libwild/Cargo.toml

```diff
@@ -48,6 +48,7 @@ uuid = { version = "1.13.1", features = ["v4"] }
 hex = "0.4.3"
 atomic-take = "1.1.0"
 normalize-path = "0.2.1"
+rustc-hash = "2.1.1"
```

Comment:
We already have ahash which should be faster (at least with AES CPU feature enabled). Any reason to bring another hashing crate?

Reply:
Good point. I used it because I had read somewhere that FxHash has better for primitive types (e.g. u32 in this case), whereas ahash was better for things like `&[u8]`. However, I just ran a benchmark (of the whole linker, not a microbenchmark) and if there's a difference, it's lost in the noise. So I've switch to ahash.